requirements.txt

```diff
@@ -1,3 +1,3 @@
-mkdocs==1.1
-mkdocs-material==4.6.3
-mkdocs-redirects==1.0.0
+mkdocs>=1.1
```

Comment:
if those versions are not working anymore should we use `>=` but with higher (currently working) versions?

Reply:
Indeed will use the current versions. This was only to verify the issue.